Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.12.3-slim
+LABEL maintainer="94nj111@gmail.com"
+
+ENV PYTHONDONTWRITEBYTECODE=True
+ENV PYTHONUNBUFFERED=True
+
+WORKDIR /app
+
+ARG UID=10001
+RUN adduser \
+    --disabled-password \
+    --gecos "" \
+    --home "/app" \
+    --shell "/sbin/nologin" \
+    --no-create-home \
```

Comment:
Remove the `--no-create-home` option when creating the `appuser` to ensure a valid home directory is available. This is important for certain applications that may require a home directory.